src/lib/types.h

```diff
@@ -294,6 +294,7 @@ typedef struct pgp_sig_subpkt_t {
         } preferred; /* 5.2.3.7.  Preferred Symmetric Algorithms */
                      /* 5.2.3.8.  Preferred Hash Algorithms */
                      /* 5.2.3.9.  Preferred Compression Algorithms */
+                     /* Crypto Refresh 5.2.3.15. Preferred AEAD Ciphersuites  */
```

Comment:
Please do not mention crypto-refresh here as it could make people guess that we officially support crypto-refresh already.